libs/core/src/main/java/com/sourcefuse/jarc/core/commonutils/CommonUtils.java

```diff
@@ -0,0 +1,27 @@
+package com.sourcefuse.jarc.core.commonutils;
```

Comment:
Can we write common-utils in java for folder names

Reply:
I guess we should use  lowecase and no special character 
description:
Using hyphens or any other special characters in package names is not allowed and will result in a compilation error. Stick to the standard naming conventions and limitations for package names to ensure compatibility and maintainability of your Java code.